docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,92 @@
+This guide was developed using a `Nokia G-240W-B` router (software version 3FE47969AGCA21). It should work for other models in the `G` series.
+
+!!! note
+    There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+
+### Grab your IPv4 and IPv6 address from your Raspberry Pi
```

Comment:
Please re-order the paragraphs in this section. It makes sense to start with "SSH into...." followed by the expected output and then everything else.

Reply:
Re-order the text, it must be visible in the next commit.